Doc/library/functions.rst

```diff
@@ -134,9 +134,9 @@ are always available.  They are listed here in alphabetical order.
    This function drops you into the debugger at the call site.  Specifically,
    it calls :func:`sys.breakpointhook`, passing ``args`` and ``kws`` straight
    through.  By default, ``sys.breakpointhook()`` calls
-   :func:`pdb.set_trace()` expecting no arguments.  In this case, it is
-   purely a convenience function so you don't have to explicitly import
-   :mod:`pdb` or type as much code to enter the debugger.  However,
+   :func:`pdb.set_trace()` expecting an optional keyword argument `header`.  In
```

Comment:
My eyes may be wrong but there might be an extra space before "In".

Reply:
In .rst files I guess there must be 2 spaces.

Reply:
ohh I see. Thanks.

Reply:
https://devguide.python.org/documenting/#use-of-whitespace

> A sentence-ending period may be followed by one or two spaces; while reST ignores the second space, it is customarily put in by some users, for example to aid Emacs’ auto-fill mode.